### ISSUE TITLE
More responsive??

### ISSUE THREAD

**Issue by naotokui** [adamstark/BTrack#3]
First of all, thank you very much for sharing this project! 

This is not an issue, but more like a general question: How can I make it more responsive to changes in tempi? Which parameter should I tweak in order to make it faster to react the change at the cost of stability or precision? (a float value called "alpha" seems to be the one??) 

Thanks in advance!

**Comment by adamstark** [comment]
Hi!

Glad you are enjoying the library. Yes, you are correct, it is the alpha parameter. By default, alpha is 0.9. This means it bases its decisions based upon 90% of its own momentum (where it expects things to be based on the idea of it being predictable) and 10% of new information. So if we dropped alpha to 0.8, the new information would have a much larger effect on the beat tracking.

The tuning of this parameter is a matter of choice - depending on whether you want responsive tracking or consistent tracking. But over tests on large databases, 0.9 was the value of alpha that produced the highest overall results, which is why we chose that.

Thanks!

Adam
